src/main/java/com/amazon/opendistroforelasticsearch/sql/antlr/OpenDistroSqlAnalyzer.java

```diff
@@ -17,28 +17,66 @@
 
 import com.amazon.opendistroforelasticsearch.sql.antlr.parser.OpenDistroSqlLexer;
 import com.amazon.opendistroforelasticsearch.sql.antlr.parser.OpenDistroSqlParser;
+import com.amazon.opendistroforelasticsearch.sql.antlr.semantic.SemanticAnalyzer;
+import com.amazon.opendistroforelasticsearch.sql.antlr.semantic.SemanticUnsupportedException;
+import com.amazon.opendistroforelasticsearch.sql.antlr.semantic.scope.SemanticContext;
 import com.amazon.opendistroforelasticsearch.sql.antlr.syntax.CaseInsensitiveCharStream;
 import com.amazon.opendistroforelasticsearch.sql.antlr.syntax.SyntaxAnalysisErrorListener;
+import com.amazon.opendistroforelasticsearch.sql.antlr.visitor.AntlrSqlParseTreeVisitor;
+import com.amazon.opendistroforelasticsearch.sql.esdomain.LocalClusterState;
 import org.antlr.v4.runtime.CommonTokenStream;
 import org.antlr.v4.runtime.Lexer;
 import org.antlr.v4.runtime.tree.ParseTree;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 
 /**
  * Entry point for ANTLR generated parser to perform strict syntax and semantic analysis.
  */
 public class OpenDistroSqlAnalyzer {
 
+    private static final Logger LOG = LogManager.getLogger();
+
+    /** Original sql query */
```

Comment:
the comment feels redundant. What about renaming field to originalSqlQuery and removing the comment?

Reply:
Sure!